apps/deploy-web/src/types/network.ts

```diff
@@ -11,4 +13,6 @@ export type Network = {
   rpcEndpoint?: string;
   version: string | null;
   enabled: boolean;
+  apiVersion: ApiVersion;
+  marketApiVersion: ApiVersion;
```

Comment:
moved these here as it's really a network config, and we have selected network definition logic based on configs already.